packages/mesos-client/index.js

```diff
@@ -0,0 +1,3 @@
+export {
+  default as createEventStreamConnection
+} from "./src/createEventStreamConnection";
```

Comment:
We need to find a good way of sharing the created stream as we shouldn't open more than one stream. Please think about an elegant interface to do so.

Reply:
I think that is out of the scope of the package. I'm not even convinced we should hardwire it with the connection manager, but making the connection a singleton out of the box will effectively merge this package with dcos ui. I planned it to be a reusable to a degree package.

Reply:
So I'd say it is a job for DCOS UI, get the connection, and then make sure it is the only one and reusable.

Reply:
Yeah, we shouldn't add the functionality to the mesos client, but we need to design a mechanism to share and manage the stream. We could introduce an IoC container to create and share the stream or build a singleton that provides an interface to subscribe to the stream. What do you think?